kandroid/src/main/kotlin/com/pawegio/kandroid/KFragment.kt

```diff
@@ -45,4 +48,12 @@ inline fun SupportFragment.toast(@StringRes resId: Int): Unit = activity.toast(r
 
 inline fun SupportFragment.longToast(@StringRes resId: Int): Unit = activity.longToast(resId)
 
-inline fun <reified T : Preference> PreferenceFragment.findPref(key: String): T = findPreference(key) as T
\ No newline at end of file
+inline fun <reified T : Preference> PreferenceFragment.findPref(key: String): T = findPreference(key) as T
+
+@TargetApi(Build.VERSION_CODES.JELLY_BEAN_MR1)
+fun Fragment.getHostingActivity(): Activity? {
+    if (activity != null) {
+        return activity
+    }
+    return parentFragment?.getHostingActivity()
```

Comment:
1. Simplify it with elvis operator: `return activity ?: parentFragment?.getHostingActivity()`
2. Convert it to `val`